src/options.cpp

```diff
@@ -104,6 +104,62 @@ zmq::options_t::options_t () :
 #endif
 }
 
+zmq::options_t::options_t (const options_t &from) :
```

Comment:
Why is this necessary? I think this should probably be avoided.

Reply:
The ctx.hpp file declares the endpoint_t struct with an instance of type options_t, and I guess the WindRiver Diab compiler isn't smart enough to auto-define the copy constructor for the options_t class (it needed the copy constructor for some reason during compilation). I can make it a VxWorks only declaration and definition.
```
struct endpoint_t
{
    socket_base_t *socket;
    options_t options;
};
```